packages/brisa/src/utils/get-page-component-with-headers/index.ts

```diff
@@ -42,6 +42,22 @@ export default async function getPageComponentWithHeaders({
     'content-type': 'text/html; charset=utf-8',
   });
 
+  pageHeaders.delete('set-cookie');
+  for (const header of [
+    middlewareResponseHeaders,
+    layoutResponseHeaders,
+    pageResponseHeaders,
+    headers,
+  ]) {
+    if (Array.isArray(header?.['set-cookie'])) {
+      for (const cookie of header['set-cookie']) {
+        pageHeaders.append('set-cookie', cookie);
+      }
+    } else if (typeof header?.['set-cookie'] === 'string') {
+      pageHeaders.append('set-cookie', header['set-cookie']);
+    }
+  }
+
```

Comment:
Why is this necessary? Without all this, you already put the cookies separated with commas, which is correct.

<img width="1299" alt="Screenshot 2025-02-13 at 22 25 31" src="https://github.com/user-attachments/assets/b339df6e-7a48-460b-b7a2-a7e82c32310c" />

Reply:
Ok I see why